tests/test_input_damages.py

```diff
@@ -73,10 +76,14 @@ def test_parse_projection_filesys(tmp_path):
     pd.testing.assert_frame_equal(df_out_expected, df_out_actual)
 
 
-def test_concatenate_damage_output(tmp_path):
+def test_concatenate_damage_output(tmp_path, monkeypatch):
     """
     Test that concatenate_damage_output correctly concatenates damages across batches and saves to a single zarr file
     """
+    monkeypatch.setattr(
+        "dscim.preprocessing.input_damages.validate_damages", lambda *args: True
+    )
+
```

Comment:
(also applies to all monkeypatching below)

Why does this need to be monkey patched out?

Might be better if the validation function takes a dataset or dataarray as input instead of doing its own IO?

Patching like this in tests might be a symptom of a design problem.